### PR TITLE
rename to wallet-legacy

### PR DESCRIPTION
**Pull request by flexsurfer**
rename to wallet-legacy

we need to make sure in review that only app-db and subscriptions were affected

### CODE REVIEW

**Review by J-Son89 — APPROVED** [review]
Thank you @flexsurfer! 🙏

File: src/status_im2/subs/root.cljs [hunk 1]
```diff
@@ -130,115 +130,6 @@
...
-(reg-root-key-sub :wallet/collectible-assets :wallet/collectible-assets)
-(reg-root-key-sub :wallet/selected-collectible :wallet/selected-collectible)
-(reg-root-key-sub :wallet/modal-selecting-source-token? :wallet/modal-selecting-source-token?)
-(reg-root-key-sub :wallet/swap-from-token :wallet/swap-from-token)
-(reg-root-key-sub :wallet/swap-to-token :wallet/swap-to-token)
-(reg-root-key-sub :wallet/swap-from-token-amount :wallet/swap-from-token-amount)
-(reg-root-key-sub :wallet/swap-to-token-amount :wallet/swap-to-token-amount)
-(reg-root-key-sub :wallet/swap-advanced-mode? :wallet/swap-advanced-mode?)
-
-;; Wallet 2
```
Comment (J-Son89) [inline src/status_im2/subs/root.cljs:193]: ;; Wallet 2
(reg-root-key-sub :wallet-2/scanned-address :wallet-2/scanned-address)

should be restored - @briansztamfater added this but I think the rebase got skewed

**Review by rasom — APPROVED** [review]

File: src/status_im2/subs/root.cljs (hunk 1 above)
Comment (J-Son89) [reply src/status_im2/subs/root.cljs:193]: looks good other than this

Comment (flexsurfer) [reply src/status_im2/subs/root.cljs:193]: good catch fixed, btw i renamed it to wallet

**Review by OmarBasem — APPROVED** [review]

**Review by smohamedjavid — APPROVED** [review]

**Comment by VolodLytvynenko** [comment]
hi @flexsurfer, thank you for the PR! Before we continue with testing, could you please describe any scenario and expected behavior introduced in this PR, or if there isn't new functionality, provide examples of areas that might be affected by this PR? Thanx!

**Comment by flexsurfer** [comment]
hey @VolodLytvynenko there is no changes in functionality, only renaming variables in the code, the affected area old wallet, thanks

**Review by briansztamfater — APPROVED** [review]
🚀

**Review by ulisesmac — APPROVED** [review]
Thanks for this PR!! :) :) :) :)

**Comment by VolodLytvynenko** [comment]
hi @flexsurfer, thank you for your work. PR is ready to be merged

**Pull request merged**